e2e/steps_file.js

```diff
@@ -51,7 +51,7 @@ module.exports = function () {
     },
 
     seeAnswerInTab(questionNo, complexTypeHeading, question, answer) {
-      const complexType = locate(`.//span[text() = '${complexTypeHeading}']`);
+      const complexType = locate(`.//span[text() = "${complexTypeHeading}"]`);
```

Comment:
Why did you make that change? Seems like irrelevant one.

Reply:
Having a complex heading of 'Social worker\'s telephone number' resulted in the line `const questionRow = locate(`${complexType}/../../../table/tbody/tr[${questionNo}]`);` having difficulties finding the DOM field. I'm suspecting because having the \' was breaking the string.